src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -73,13 +73,22 @@ def validate_parameters(ctx, param, value):
 
 @cli_bundle.command(name="submit")
 @click.option(
-    "-p", "--parameter", multiple=True, callback=validate_parameters, help="Job template parameters"
+    "-p",
+    "--parameter",
+    multiple=True,
+    callback=validate_parameters,
+    help='The values for the job template\'s parameters. Can be provided as key-value pairs, inline JSON strings, or as paths to a JSON or YAML document. If provided more than once, the values are combined in the order that they appear. Examples: --parameter MyParam=5 -p file://parameter_file.json -p \'{"MyParam": "5"}\'',
 )
 @click.option("--profile", help="The AWS profile to use.")
 @click.option("--farm-id", help="The AWS Deadline Cloud Farm to use.")
 @click.option("--queue-id", help="The AWS Deadline Cloud Queue to use.")
 @click.option("--name", help="The job name to use in place of the one in the job bundle.")
-@click.option("--priority", type=int, default=50, help="The priority of the job.")
+@click.option(
+    "--priority",
+    type=int,
+    default=50,
+    help="The priority of the job. The highest priority is 100.",
```

Comment:
What do you think about clarifying whether jobs with a higher or lower number are scheduled first?

Reply:
actually kind of surprised we're not enforcing validation at the command level. Rather than when it gets to the service... Food for thought, may make sense to add it here.

Reply:
I copied this from our API docs for indicating which number represents higher priority. But agree it could be clearer, will fix.

If someone gives a priority that's out of range, they'll quickly get back an error from the boto3 client even before the API is made, so I don't think we _need_ to mirror those validations here. I also think the help text gets a little noisy with the extra validations added. Example excerpt:
```
  --priority INTEGER RANGE        The priority of the job. Jobs with a higher
                                  priority run first.  [0<=x<=100]
```